sdk/iOS/src/MSQueuePushOperation.m

```diff
@@ -225,6 +225,43 @@ - (void) finishTableOperation:(MSTableOperation *)operation item:(NSDictionary *
                                               internalError:storeError];
                 }
             }
+            else {
+                // Operation was successful, but another operation is pending.
```

Comment:
This is a logical place to handle a conflicts during a push with a saved op.  That doesn't help anything already in memory in the app. 

I'd like to see the logic I outlined as the fallback case I think either way.

If adding a conflict handling delegate method it would ideally be usable from a SyncTable action, and easily called from the tableOperation: onComplete: method as well as here.  Also, potentially from a item change notification during Pulls & Pushes.